### DISCUSSION TITLE
Support for Command Completion

### DISCUSSION THREAD

**Discussion by FlorianWDP** [spectreconsole/spectre.console#1149] [Q&A] [answered]
I was wondering if there is an in-built functionality for tab completion of commands. As a reference, i would like to do something similar to how git handles tab completion, but i would already be satisfied if i could tab complete a started command.

**Comment by nils-a** [comment, marked as answer]
Currently, there is none.
The corresponding issue is https://github.com/spectreconsole/spectre.console/issues/267 and there is some work in progress at https://github.com/spectreconsole/spectre.console/pull/727
